angular-workspace/projects/ni/nimble-angular/src/directives/combobox/nimble-combobox-control-value-accessor.directive.ts

```diff
@@ -131,7 +131,10 @@ export class NimbleComboboxControlValueAccessorDirective implements ControlValue
         } else {
             this._displayTextToOptionsMap.set(option.text, [option]);
         }
-        this.updateDisplayValue();
+
```

Comment:
Wouldn't we also unrevert this change?
https://github.com/ni/nimble/pull/1949

Maybe it's worth testing tgz builds of both changes together in the end application to make sure a regression is not re-introduced?

Reply:
Yes, I was just going to do that as a separate PR. I don't mind doing them both at the same time though. I'll test a tgz build with both changes.

Reply:
I've now "unreverted" the change to the `nimble-components` `Combobox`. I've tested the tgz builds in the application with the noted regression and saw no performance change.